### PR TITLE
Update search-sku-tier.md

### PR DESCRIPTION
**Pull request by angiesi**
Adding suggested alternatives

### CODE REVIEW

**Comment by Jak-MS** [comment]
@HeidiSteen 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/search/search-sku-tier.md [hunk 1]
```diff
@@ -55,19 +55,17 @@ You can find out more about the various tiers on the [pricing page](https://azur
...
-| West Europe | All tiers |
-| West US 3| All tiers |
+Currently, several regions are at capacity for specific tiers and can't be used for new search services without impacting existing services. If you use the Azure portal to create a search service, the portal excludes any region-tier combinations that aren't available.
+
+| Region | Disabled tier (SKU) due to over-capacity | Suggested Alternatives  |
+|--------|------------------------------------------|-------------------------|
+| Central India | S2, S3, S3HD, L1, L2| South India |
+| East US| All tiers| Central US|
+| Qatar Central | All tiers| UAE North |
+| South Central US | All tiers | Central Us |
```
Comment (HeidiSteen) [inline articles/search/search-sku-tier.md]: Central US

File: articles/search/search-sku-tier.md [hunk 2]
```diff
@@ -55,19 +55,17 @@ You can find out more about the various tiers on the [pricing page](https://azur
...
-| Central US | S2, S3, S3HD, L1, L2 |
-| Central India | S2, S3, S3HD, L1, L2|
-| East US| All tiers|
-| Japan East | S2, S3, S3HD, L1, L2 |
-| Qatar Central | All tiers|
-| South Central US | All tiers |
-| US Gov Virginia | All tiers |
-| West Europe | All tiers |
-| West US 3| All tiers |
+Currently, several regions are at capacity for specific tiers and can't be used for new search services without impacting existing services. If you use the Azure portal to create a search service, the portal excludes any region-tier combinations that aren't available.
```
Comment (HeidiSteen) [inline articles/search/search-sku-tier.md:58]: I think the extra clause "without impacting existing services" adds complexity but minimal value because it's common sense that you would stop allowing new services if it was going to destabilize the whole cluster.   In general, it's better to write succinctly because people read fast, and if content is too wordy they start skipping around.

**Review by HeidiSteen — CHANGES_REQUESTED** [review]
Typo and suggestion

**Comment by HeidiSteen** [comment]
#please-close  The article in question was updated in other PRs.  This PR would overwrite newer content with stale content.

**Comment by angiesi** [comment]
[heart]         Angie Silva Pereyra reacted to your message:
________________________________
